### PR TITLE
DIT 963 - Referral Report

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/models/ReportResult.scala [hunk 1]
```diff
@@ -20,4 +20,7 @@ case class ReportResult
 (
   group: Option[String],
   value: Seq[Int]
-)
+) {
+  def size: Int = value.size
+  def average: Double = value.sum.toDouble / size
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/tariffclassificationfrontend/models/ReportResult.scala]: Do we need to guard against size being zero?

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/ReferralReport.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+package uk.gov.hmrc.tariffclassificationfrontend.views
+
+import uk.gov.hmrc.tariffclassificationfrontend.models.{Queue, ReportResult}
+
+class ReferralReport(results: Seq[ReportResult]) {
+
+  lazy val count: Int = results.map(_.size).sum
+
+  lazy val average: Int = Math.round(results.flatMap(_.value).sum.toDouble / count).toInt
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/ReferralReport.scala]: Again, can count be zero?

File: app/uk/gov/hmrc/tariffclassificationfrontend/models/ReportResult.scala (hunk 1 above)
Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/models/ReportResult.scala]: Count will never be zero :) there wouldnt be a ReportResult for it if it was

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/ReferralReport.scala (hunk 2 above)
Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/ReferralReport.scala]: Count will never be zero :) there wouldnt be a ReportResult for it if it was

Comment (SteveSugden) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/ReferralReport.scala]: I thought that'd be the case, but worth asking the question.

**Pull request merged**